yt_dlp/extractor/nuum.py

```diff
@@ -0,0 +1,133 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    parse_iso8601,
+    traverse_obj,
+    try_get,
+)
+
+
+class NuumBaseIE(InfoExtractor):
+
+    def _fetch(self, path, video_id, description, query={}):
+        response = self._download_json(
+            f'https://nuum.ru/api/v2/{path}', video_id, query=query,
+            note=f'Downloading {description} metadata',
+            errnote=f'Unable to download {description} metadata')
+        error = response.get('error')
+        if error:
+            raise ExtractorError(f'{self.IE_NAME} returned error: {error}', expected=True)
```

Comment:
Is this really expected?
```suggestion
            raise ExtractorError(f'API returned error: {error!r}')
```